extensions/acl_wl/aw_conf.l

```diff
@@ -86,7 +86,8 @@ ALLOW_OLD_TLS		{
 			}
 
 	/* Any other string is considered a fqdn or partial fqdn with a star. The star can only be the first label. */
-(\*|[[:alnum:]][[:alnum:]-]*)(\.[[:alnum:]][[:alnum:]-]*)+	{
+(\*|[[:alnum:]][[:alnum:]-]*)(\.[[:alnum:]][[:alnum:]-]*)+ |
```

Comment:
I think there might be more concise options (`\*?[[:alnum:]][[:alnum:]-]*`) but I'm not too bothered.